File: notification-app/bot-telegram.js
const TelegramBot = require('node-telegram-bot-api');
const token = '5316988190:AAHMLldZf6GafRq_TXpQ22gddmDQnjE6b3w';
const bot = new TelegramBot(token, {polling: true});
const text = "O arquivo foi rotacionado e foi transformado em escala de cinza"


bot.sendMessage(957366775, text)

File: bot-telegram.py
import telegram

api_key = '5316988190:AAHMLldZf6GafRq_TXpQ22gddmDQnjE6b3w'
user_id = '957366775'

bot = telegram.Bot(token=api_key)

bot.send_message(chat_id=user_id, text='Está funcionando')
